core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -153,19 +164,22 @@ def validate_locations(workbook: dict[str, pd.DataFrame]) -> list["TownsFundRoun
     single_rows = project_details_df[project_details_df["Single or Multiple Locations"] == MultiplicityEnum.SINGLE]
     multiple_rows = project_details_df[project_details_df["Single or Multiple Locations"] == MultiplicityEnum.MULTIPLE]
 
-    # tuples of (column_data, column_name) to validate empty cells on
+    # tuples of (column, column_data, column_name) to validate empty cells on
```

Comment:
can this naming be improved? what is `column` vs `column_name`? Do we need both? Seems convoluted

Reply:
I agree, the two here mean internal table column and form column. we don'thave an existing mapping that works perfectly for this use case but I can try to make it more clear